examples/1.0.0/bnpl-arazzo.yaml

```diff
@@ -0,0 +1,252 @@
+          oneOf:
+          - type: object
+            required:
+            - firstName
+            - lastName
+            - dateOfBirth
+            - postalCode
+            properties:
+              firstName:
+                description: First name of customer
+                type: string
+                minLength: 1
+                maxLength: 70
+              lastName:
+                description: Last name of customer
+                type: string
+                minLength: 1
+                maxLength: 70
+              dateOfBirth:
+                description: Customer date of birth
+                type: string
+                format: date-time
+              postalCode:
+                description: Zip code or postal code of customer
+                type: string
+                minLength: 1
+                maxLength: 70
+              additionalProperties: false
+          - type: object
+            required:
+            - uri
+            properties:
+              uri:
+                description: URI that points to an existing customer resource, as customer already enrolled on platform
+                type: string
+                format: uri
+            additionalProperties: false
+        products:
+          type: array
+          minItems: 1
+          items:
+            type: object
+            required:
+            - productCode
+            - purchaseAmount
+            properties:
+              merchantCategoryCode:
+                description: Merchant category code of merchant. Only required for marketplace ecommerce platforms
+                type: string
+                pattern: '^[0-9]{4}$'
+              productCode:
+                description: Product code for loan application. Required for eligibility check
+                type: string
+              purchaseAmount:
+                description: Product purchase amount and currency code
+                type: object
+                required:
+                  - currency
+                  - amount
+                properties:
+                  currency:
+                    description: Currency code
+                    type: string
+                    pattern: "^[A-Z]{3}$"
+                  amount:
+                    description: Amount
+                    type: number
+  steps:
+  - stepId: checkLoanCanBeProvided
+    description: |
+      Call the BNPL API to filter the basket for products qualifying for checkout loans. Pass in the array of products from the workflow input as the payload for the API call.
+
+      Act on the response payload:
+      
+      - If a list of qualifying products is returned then submit customer choices.
+      - If the list of qualifying products is empty then end the workflow
+    operationId: findEligibleProducts
+    parameters:
+      - name: products
```

Comment:
To send a request body to an API operation, you should leverage the [Request Body Object](https://github.com/OAI/Arazzo-Specification/blob/main/versions/1.0.0.md#request-body-object). Sending a body within parameters is no longer supported.

FYI this is applicable across multiple of the steps in this Arazzo document.